Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -0,0 +1,78 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _kubeproxy-free:
+
+************************************
+Kubernetes without kube-proxy (beta)
+************************************
+
+This guide explains how to provision a Kubernetes cluster without
+``kube-proxy``, and to use Cilium to replace it. For the simplicity
+reason we will use ``kubeadm`` to bootstrap the cluster.
+
```

Comment:
Could we split this into two?

First part, clearly marked as experimental which would outline the workflow based on your local branch (including how to clone+build) where kubeadm natively does not provision kube-proxy. Also mentioning that as soon as this is merged upstream, we are updating the GSG in here.

Second part, the below guide for working around the issue where kubeadm cannot be updated.

Reply:
Thanks, updated as discussed offline.